### PR TITLE
[JHipster 4] Promote Yarn as default installation

### PR DESCRIPTION
**Pull request by pascalgrimaud**

### CODE REVIEW

File: pages/creating_an_app.md [hunk 1]
```diff
@@ -163,7 +163,9 @@ Here are the options you can pass:
 * `--skip-user-management` - Skip the user management generation, both on the back-end and on the front-end (Default: false)
 * `--i18n` - Disable or enable i18n when skipping client side generation, has no effect otherwise (Default: true)
 * `--with-entities` - Regenerate the existing entities if they were already generated (using their configuration in the `.jhipster` folder) (Default: false)
-* `--check-install` - Check your installation is correct (Default: true)
+* `--skip-checks` - Check the status of the required tools (Default: false)
```
Comment (jdubois) [inline pages/creating_an_app.md]: Rather "Skip the check" than "Check the status", no?

File: pages/installation.md [hunk 2]
```diff
@@ -15,43 +15,49 @@ sitemap:
...
-5.  (Recommended) Update npm: `npm install -g npm`
-6.  Install Yeoman: `npm install -g yo`
-7.  Install Bower: `npm install -g bower`
-8.  Install Gulp: `npm install -g gulp-cli` (If you have previously installed a version of gulp globally, please run `npm rm -g gulp` to make sure your old version doesn't collide with `gulp-cli`)
-9.  Install JHipster: `npm install -g generator-jhipster`
-10.  (Optional) Install Yarn: `npm install -g yarn` (So you can use the yarn flag `yo jhipster --yarn`)
+4.  Install Node.js from [the Node.js website](http://nodejs.org/) (prefer an LTS version)
+5.  Install Yarn from [the Yarn website](https://yarnpkg.com/en/docs/install)
+6.  Install Yeoman: `yarn global add yo`
+7.  Install Bower (only for Angular1): `yarn global add bower`
```
Comment (jdubois) [inline pages/installation.md]: I would write it the opposite way:
- Only for AngularJS 1: Install Bower...

File: pages/installation.md [hunk 3]
```diff
@@ -15,43 +15,49 @@ sitemap:
...
+
+On Mac or Linux: ```export PATH="$PATH:`yarn global bin`:$HOME/.config/yarn/global/node_modules/.bin"```
 
-To find more information, tips and help, please have a look at [the Yeoman "getting starting" guide](http://yeoman.io/learning/index.html) and at the [NPM documentation](https://docs.npmjs.com/) before [submitting a bug](https://github.com/jhipster/generator-jhipster/issues?state=open).
+To find more information, tips and help, please have a look at [the Yeoman "getting starting" guide](http://yeoman.io/learning/index.html) and at the [Yarn documentation](https://yarnpkg.com/) before [submitting a bug](https://github.com/jhipster/generator-jhipster/issues?state=open).
 
 Now that JHipster is installed, your next step is to [create an application]({{ site.url }}/creating-an-app/)
 
-## [Beta] Local installation with Yarn
+## Local installation with npm (alternative to Yarn)
```
Comment (jdubois) [inline pages/installation.md]: I think NPM should be in uppercase

**Comment by pascalgrimaud** [comment]
Done. Thanks for reviewing @jdubois !

**Pull request merged**